iostat/command/command.go

```diff
@@ -35,3 +35,9 @@ func (c *cmdRunner) Run(cmd string, args []string) (io.Reader, error) {
 		return nil, fmt.Errorf("time out (cmd:%v args:%v)", cmd, args)
 	}
 }
+
+func (c *cmdRunner) Exec(cmd string, args []string) string {
+	command := exec.Command(cmd, args...)
+	outputBytes, _ := command.CombinedOutput()
```

Comment:
When I was doing it, I had same problems on centos:6, because executing "iostat -V" returns version info on stdout, and error on stderr (on ubuntu:xenial only stdout return). That's why I don't handle this error.
Should I change that method, to take 0 args and return version string of iostat?